cluster/gce/gci/configure-helper.sh

```diff
@@ -1731,7 +1731,9 @@ function prepare-konnectivity-server-manifest {
 
   params+=("--server-port=0")
   params+=("--agent-port=$1")
-  params+=("--admin-port=$2")
+
+  params+=("--health-port=$2")
```

Comment:
The documentation for prepare-konnectivity-server-manifest is now wrong. Can we please fix it?
# $2: value for variable "health_port"
# $3: value for variable "admin_port"

Reply:
Nice catch. Fixed.